backend/recipes_seed.sql

```diff
@@ -0,0 +1,1463 @@
+//GREENER VILLAGE RECIPES FOR THE TABLE MEAL:
```

Comment:
Comment character is -- for postgres and not // 
Didn't it give an error while testing the sql?